### ISSUE TITLE
Change tests so different backends are not tested in the same test

### ISSUE THREAD

**Issue by zerothi** [dtudk/dtumathtools#13]
One should use `pytest.mark.parametrize` instead, here we can even allow `importorskip`.
This would be more stable and easier to test going forward.

**Comment by Chrillebon** [comment]
Agreed. However, not all tests will work like this, as the `kwargs` will be sent to the specific backend.

**Comment by zerothi** [comment]
I don't know what you mean?

This would work just fine:

```python
@pytest.mark.parametrize("backend", [
MP, pytest.param(PB, pytest.mark.skipif(... plotly importable)])
def test_quiver(backend):
    dtuplot.quiver(..., backend=backend)
```
with some changes to make it fully functional.

**Comment by zerothi** [comment]
There are many ways to expand on this.
But the idea would be to explicitly create tests for each backend. Having everything in 1 is not ideal.

**Comment by Chrillebon** [comment]
Agreed! And the addition of the skipif not importable is a great way of getting around the challenges with not every backend being installed as default!
